yarn.lock

```diff
@@ -11438,6 +11469,23 @@ osmtogeojson@^3.0.0-beta.4:
   optionalDependencies:
     "@types/geojson" "^1.0.2"
 
+osmtogeojson@^3.0.0-beta.5:
```

Comment:
beta.4 is still fetched because it is in the transition-common's package.json. It does not appear used there. could you remove it from there?